earth_enterprise/src/common/khxml/khxml.cpp

```diff
@@ -191,7 +278,7 @@ bool
 WriteDocumentImpl(DOMDocument *doc, const std::string &filename) throw()
 {
 
-  InitializeXMLLibrary();
+  //InitializeXMLLibrary();
```

Comment:
Is it guaranteed that the XML library will be initialized before this function is called?